src/pages/About/About.stories.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import About from './About';
+
+export default {
+  title: 'Components/Pages/About',
```

Comment:
How'd you feel about this being 'Pages/About' instead of 'Components/Pages/About', so that they're grouped separately in Storybook (and it mirrors the directory structure)?

Same goes for all the other pages, of course.